node/Dockerfile

```diff
@@ -7,3 +7,5 @@ FROM node:alpine
 #   && apk add nodejs@edge \
 #   && npm uninstall -g npm \
 #   && rm -rf /var/cache/apk/*
+
+RUN addgroup -g 1000 -S fn && adduser -S -u 1000 -G fn fn
```

Comment:
I understand what this is supposed to do and it looks good. Did you 'test' it by spinning up a container, exec-ing in and checking it? If so I will approve.

Reply:
i think this is the second round of testing